### PR TITLE
use git polkadot-sdk deps

### PR DESCRIPTION
**Pull request by pgherveou**

### CODE REVIEW

File: node/Cargo.toml [hunk 1]
```diff
@@ -6,8 +6,8 @@ description = "Substrate node configured for smart contracts via `pallet-contrac
 edition = "2021"
 license = "Unlicense"
 build = "build.rs"
-homepage = "https://github.com/paritytech/substrate-contracts-node"
-repository = "https://github.com/paritytech/substrate-contracts-node"
+homepage = "https://github.com/paritytech/polkadot-sdk-contracts-node"
+repository = "https://github.com/paritytech/polkadot-sdk-contracts-node"
```
Comment (pgherveou) [inline node/Cargo.toml:10]:
```suggestion
homepage = "https://github.com/paritytech/substrate-contracts-node"
repository = "https://github.com/paritytech/substrate-contracts-node"
```

File: node/Cargo.toml
Comment (pgherveou) [inline node/Cargo.toml:1]: I probably need to mark it as publish = false if we decide to keep git ref

**Review by ascjones — COMMENTED** [review]
All git deps should be more spcific

**Comment by pgherveou** [comment]
You mean we should use a tag? Isn't relying on the lock file more convenient to update to newer version of the code?

File: runtime/Cargo.toml [hunk 2]
```diff
@@ -5,8 +5,8 @@ authors = ["Parity Technologies <admin@parity.io>"]
 description = "Runtime configured for smart contracts via `pallet-contracts`."
 edition = "2021"
 license = "Unlicense"
-homepage = "https://github.com/paritytech/substrate-contracts-node"
-repository = "https://github.com/paritytech/substrate-contracts-node"
+homepage = "https://github.com/paritytech/polkadot-sdk-contracts-node"
+repository = "https://github.com/paritytech/polkadot-sdk-contracts-node"
```
Comment (pgherveou) [inline runtime/Cargo.toml:9]:
```suggestion
homepage = "https://github.com/paritytech/substrate-contracts-node"
repository = "https://github.com/paritytech/substrate-contracts-node"
```

**Comment by ascjones** [comment]
> You mean we should use a tag? Isn't relying on the lock file more convenient to update to newer version of the code?

Better to be explicit, we can't guarantee someone will install using `--locked`. It's easy enough do do a search/replace with `branch = ""` We used to track the `polkadot-*` branches. I see there is now e.g. https://github.com/paritytech/polkadot-sdk/tree/release-polkadot-v1.2.0

**Comment by pgherveou** [comment]
Sounds good will do that. Thanks for the feedback Andrew

**Comment by pgherveou** [comment]
@ascjones updated to use `branch = "release-polkadot-v1.2.0"` everywhere

**Review by ascjones — APPROVED** [review]
LGTM, though we must remember to switch back to crates.io dependencies as soon as they are available.

**Comment by pgherveou** [comment]
I should probably use this branch to check that ink e2e tests are still passing before merging right? Anything else you do usually ?

**Comment by ascjones** [comment]
> I should probably use this branch to check that ink e2e tests are still passing before merging right? Anything else you do usually ?

Yes that's a good idea to do that, you can either do that here or as part releasing the next version with these changes.

**Comment by pgherveou** [comment]
found some issues with transfer that has to be renamed to transfer_allow_death in ink e2e test.
Will merge this and push a PR to ink

**Pull request merged**